### ISSUE TITLE
plugin.json issues

### ISSUE THREAD

**Issue by cschol** [thomassidor/tinytricks#1]
```
MG1: invalid module tags: low-frequency oscilator
MG8: invalid module tags: low-frequency oscilator
MG16: invalid module tags: low-frequency oscilator
-- Valid tags are defined in https://raw.githubusercontent.com/VCVRack/Rack/v1/src/tag.cpp
Invalid URL: https://thomassidor.com
```

**Comment by thomassidor** [comment]
@cschol Could you elaborate what's wrong with the author URL? It works fine for me.

**Comment by thomassidor** [comment]
Tags fixed in commit 725d63321b13fff18b7620a62ad13ba4e0ae1804 to master.

**Comment by thomassidor** [comment]
Tags fixed in v1.1.0 - closing.
